packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -4,12 +4,27 @@
 
 import 'dart:async' show Future;
 import 'dart:typed_data' show Uint8List;
-import 'dart:ui' show Size;
 
 import 'package:flutter/foundation.dart' show kIsWeb;
-import 'package:flutter/material.dart'
-    show ImageConfiguration, AssetImage, AssetBundleImageKey;
-import 'package:flutter/services.dart' show AssetBundle;
+import 'package:flutter/widgets.dart'
+    show
+        AssetBundle,
+        AssetBundleImageKey,
+        AssetImage,
+        ImageConfiguration,
+        Size,
+        WidgetsBinding;
+
+/// Type of bitmap scaling to use on BitmapDescriptor creation.
+enum BitmapScaling {
+  /// Automatically scale image with devices pixel ratio or to given size,
+  /// to keep marker sizes same between platforms and devices.
+  auto,
+
+  /// Render marker to the map as without scaling, this can be used if the image
```

Comment:
Nit: Replace the `,` after scaling with a `.` and two newlines (and then capitalize `This`).